app/classifier/components/TaskNavButtons/components/TalkLink/TalkLink.jsx

```diff
@@ -21,39 +23,78 @@ const commonStyles = `
   position: relative;
   text-align: center;
   text-decoration: none;
+  white-space: nowrap;
 `;
 
 export const StyledTalkLink = styled(Link)`
   ${commonStyles}
+  background: ${theme('mode', {
+    dark: zooTheme.colors.darkTheme.background.default,
+    light: TALK_LINK_BLUE
+  })};
+  border: ${theme('mode', {
+    dark: `thin solid ${TALK_LINK_BLUE}`,
+    light: 'thin solid transparent'
+  })};
+  color: ${theme('mode', {
+    dark: zooTheme.colors.darkTheme.font,
+    light: 'white'
+  })};
 
   &:hover, &:focus {
-    background: #69c9fd;
+    background: ${theme('mode', {
+      dark: TALK_LINK_BLUE_HOVER,
+      light: TALK_LINK_BLUE_HOVER
+    })};
+    border: ${theme('mode', {
+      dark: `thin solid ${TALK_LINK_BLUE}`,
+      light: `thin solid ${TALK_LINK_BLUE_HOVER}`
+    })};
   }
 `;
 
-export const StyledDisabledTalkPlaceholder = styled.span`
+// can't style the cursor and pointer-events: none simulatenously
+export const StyledDisabledTalkPlaceholder = styled.span.attrs({
+  tabIndex: -1
```

Comment:
tabIndex only applies to elements that can receive focus. Spans can't receive focus so this isn't necessary. Sorry, my original comment was about disabling links. Spans don't receive pointer events by default either, so I'm not sure why the span would use `pointer-events: none`.

Reply:
My fault for not reading the code properly in my first review.